modules/deployment/pom.xml

```diff
@@ -85,7 +85,7 @@
             <plugin>
                 <groupId>org.openapitools</groupId>
                 <artifactId>openapi-generator-maven-plugin</artifactId>
-                <version>6.6.0</version>
+                <version>6.4.0</version>
```

Comment:
why this change?

Reply:
The classes generated in version 6.6.0 do not have @ Component annotations, only in version 6.4.0. The client we need to generate is a spring bean.

Reply:
@jinyangyang222 can you keep the 6.6.0 version and add "generateClientAsBean" config as true? Pls see if this works.

Reply:
@iskey fixed， @swaroopar thank you very much, it's ok.